drop-transfer/src/event.rs

```diff
@@ -28,7 +28,7 @@ pub enum Event {
     FileUploadFailed(Transfer, FileId, Error),
     FileDownloadFailed(Transfer, FileId, Error),
 
-    TransferCanceled(Transfer, bool),
+    TransferCanceled(Transfer, bool, bool),
 
-    TransferFailed(Transfer, Error),
+    TransferFailed(Transfer, Error, bool),
```

Comment:
Bool is not a very descriptive name as it depends on the context. For such case I would recommend introducing an enum as it reduces guesswork

Reply:
done